<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>New PTNG Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/workbench.qrc">
    <normaloff>:/images/goldings_gym_core_00.png</normaloff>:/images/goldings_gym_core_00.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Display name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>9</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Project file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFile">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>85</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>83</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Select project file name</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/folder_open_01.png</normaloff>:/images/folder_open_01.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="fileWatchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>132</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>File watch directories:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileWatchPushButton">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>132</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Add file watch directory</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/folder_open_01.png</normaloff>:/images/folder_open_01.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFileWatchPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>132</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Remove file watch directory</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/close_01.png</normaloff>:/images/close_01.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="fileWatchListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>382</width>
     <height>136</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>156</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>45</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Id:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditId</tabstop>
  <tabstop>lineEditFile</tabstop>
  <tabstop>pushButtonFile</tabstop>
  <tabstop>addFileWatchPushButton</tabstop>
  <tabstop>removeFileWatchPushButton</tabstop>
  <tabstop>fileWatchListWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../res/workbench.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
